pycm/pycm_util.py

```diff
@@ -206,23 +206,22 @@ def normalized_table_calc(classes, table):
         div = sum(table[key].values())
         if div == 0:
             div = 1
+        p = float(10**5)
```

Comment:
Why here ?!?
Out of any **for loop** ;-)

Reply:
Oh, you're right, just a mistake taken due to hurry.